torch/_export/constraints.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+import warnings
+
+import torch._dynamo
+from torch._dynamo.comptime import comptime
+from torch.fx.experimental.symbolic_shapes import constrain_range
+
+
+def add_inline_constraint(symbol, min: Optional[int] = None, max: Optional[int] = None):
```

Comment:
Can you leave TODO here that we want to hide this min/max stuff under DynamicVar or some kind of abstraction that is similar to DynamicDim.